repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -318,6 +318,10 @@ object HttpServletRequestWrapper {
     parsedParameterMap.toMap
   }
 
+  def parseQueryString(queryString: String): java.util.Map[String, Array[String]] = {
```

Comment:
What am I even talking about? So ignore that bit about it being a companion object function, which is clearly is.

I guess the rest is still a thing that could be done.